examples/l1-resources/federated-database-instance.ts

```diff
@@ -0,0 +1,71 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnFederatedSettingsOrgRoleMapping } from 'awscdk-resources-mongodbatlas';
+
+interface AtlasStackProps {
+  readonly tenantName: string;
+  readonly projId: string;
+  readonly profile: string;
+  readonly roleId: string;
+  readonly testS3Bucket: string;
+}
+
+export class CDKFederatedDatabaseInstanceExample extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const atlasProps = this.getContextProps();
+    const federatedDatabaseInstance = new CfnFederatedSettingsOrgRoleMapping(this, 'FederatedDatabaseInstance', {
+      tenantName: atlasProps.tenantName,
+      projectId: atlasProps.projId,
+      profile: atlasProps.profile,
+      cloudProviderConfig: {
+        roleId: atlasProps.roleId,
+        testS3Bucket: atlasProps.testS3Bucket,
+      },
+      dataProcessRegion: {
+        region: "VIRGINIA_USA",
```

Comment:
curious why this is fixed?

Reply:
is an example, some values are fixed as default to make it easier for the user

Reply:
whops I missed the `examples` folder